source/docs/v3/integrations/sanctum.blade.md

```diff
@@ -6,17 +6,33 @@ section: content
 
 # Laravel Sanctum {#sanctum}
 
-> Note that the `sanctum` auth guard can't be used with [user impersonation]({{ $page->link('features/user-impersonation') }}) because user impersonation supports stateful guards only.
+> Note: The `sanctum` auth guard can't be used with [user impersonation]({{ $page->link('features/user-impersonation') }}) because user impersonation supports stateful guards only.
 
-If you need to use the `csrf-cookie` route that Sanctum provides, you have to set up [universal routes]({{ $page->link('features/universal-routes') }}) in your app. Then, add `'routes' => false` to the `sanctum.php` config.
+### Sanctum's csrf-cookie route in the tenant app
```

Comment:
The page now has two sections about `csrf-cookie` routes, but no introduction. Maybe add a brief paragraph saying that Sanctum works out of the box, but if you want to use the `csrf-cookie` route, you need to make this small change? (If I understand the integration correctly).

Reply:
@stancl is this fine? https://github.com/stancl/tenancy-docs/pull/227/commits/05091c1a39c32563e3cb5e1947522debf310aca8

Reply:
Yeah, looks good. The PR is finished now I think?

Reply:
I think so👍